SPECS/kernel/config

```diff
@@ -68,7 +68,9 @@ CONFIG_GENERIC_IRQ_EFFECTIVE_AFF_MASK=y
 CONFIG_GENERIC_PENDING_IRQ=y
 CONFIG_GENERIC_IRQ_MIGRATION=y
 CONFIG_HARDIRQS_SW_RESEND=y
+CONFIG_GENERIC_IRQ_CHIP=y
```

Comment:
https://www.kernel.org/doc/html/v5.0/core-api/genericirq.html#:~:text=Linux%20generic%20IRQ%20handling%20Introduction%20The%20generic%20interrupt,all%20the%20different%20types%20of%20interrupt%20controller%20hardware.

Reply:
This creates an generic interface for iqrs